tests/collections/common/test_lhotse_dataloading.py

```diff
@@ -37,16 +37,18 @@
 
 @pytest.fixture(scope="session")
 def cutset_path(tmp_path_factory) -> Path:
-    """10 utterances of length 1s as a Lhotse CutSet."""
+    """12 utterances of length 1s as a Lhotse CutSet."""
```

Comment:
Could you avoid modifying existing fixtures and create a new one, dedicated specifically to this feature? There are many tests depending on those, I don't want to further increase test setup complexity.

To clarify: my point is that with this change, every test in this file is testing the "skipme" functionality. It is simpler to debug if there is just one test that's testing it instead.